Cargo.toml

```diff
@@ -5,4 +5,8 @@ authors = ["Aaron Kofsky <a2aarontothe2@gmail.com>"]
 
 [dependencies]
 rand = "0.3"
-fs2 = "0.4"
\ No newline at end of file
+fs2 = "0.4"
+egg-mode = "0.11.0"
+dotenv = "0.10.1"
+futures = "0.1"
```

Comment:
We don't need futures unless we want to do async stuff. Since we're just planning to post a tweet once an hour, it'd be good to remove it.